docs/integrations/destinations/teradata.md

```diff
@@ -0,0 +1,88 @@
+* **JDBC URL Params** (optional)
+
+[Refer to this guide for more details](https://downloads.teradata.com/doc/connectivity/jdbc/reference/current/jdbcug_chapter_2.html#BGBHDDGB)
+
+## Sync overview
+
+### Output schema
+
+Each stream will be output into its own table in Teradata. Each table will contain 3 columns:
+
+* `_airbyte_ab_id`: a uuid assigned by Airbyte to each event that is processed. The column type in Teradata is `VARCHAR(256)`.
+* `_airbyte_emitted_at`: a timestamp representing when the event was pulled from the data source. The column type in Teradata is `TIMESTAMP(6)`.
+* `_airbyte_data`: a json blob representing with the event data. The column type in Teradata is `JSON`.
+
+
+### Features
+
+The Teradata destination connector supports the
+following[ sync modes](https://docs.airbyte.com/cloud/core-concepts#connection-sync-modes):
+
+
+| Feature | Supported?\(Yes/No\) | Notes |
+| :--- | :--- | :--- |
+| Full Refresh Sync | Yes |  |
+| Incremental - Append Sync | Yes |  |
+| Incremental - Deduped History | No |  |
+| Namespaces | Yes |  |
+
+### Performance considerations
+
+## Getting started
+
+### Requirements
+
+You need a Teradata user with the following permissions:
+
+* can create tables and write permission.
+* can create schemas e.g:
+
+You can create such a user by running:
+
+```
+CREATE USER airbyte_user  as perm=10e6, PASSWORD=<password>;
+GRANT ALL on dbc to airbyte_user;
+
+```
+
+You can also use a pre-existing user but we highly recommend creating a dedicated user for Airbyte.
+### Setup guide
+
+#### Set up the Teradata Destination connector
+
+1. Log into your Airbyte Open Source account.
+2. Click **Destinations** and then click **+ New destination**.
+3. On the Set up the destination page, select **Teradata** from the **Destination type** dropdown.
+4. Enter the **Name** for the Teradata connector.
+5. For **Host**, enter the host domain of the Teradata instance
+6. For **Default Schema**, enter the Default Schema name. The default value is public.
+7. For **User** and **Password**, enter the database username and password.
+8. To customize the JDBC connection beyond common options, specify additional supported [JDBC URL parameters](https://downloads.teradata.com/doc/connectivity/jdbc/reference/current/jdbcug_chapter_2.html#BGBHDDGB) as key-value pairs separated by the symbol & in the **JDBC URL Params** field.
+
+   Example: key1=value1&key2=value2&key3=value3
+
+   These parameters will be added at the end of the JDBC URL that the AirByte will use to connect to your Teradata database.
+
+## CHANGELOG
+
+| Version | Date       | Pull Request                                                  | Subject                          |
+|:--------|:-----------|:--------------------------------------------------------------|:---------------------------------|
+| 0.1.0   | 2022-12-13 | https://github.com/airbytehq/airbyte/pull/20428			   | New Destination Teradata Vantage |
```

Comment:
@marcosmarxm can we use the 0.0.1 as initial version of teradata vantage connector instead of 0.1.0?

Reply:
The Airbyte default is 0.1.0